client/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Canteen</title>
+  <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" />
```

Comment:
I'll switch to bower and a lib file in the client after lunch

On Thursday, April 7, 2016, Ryan D Haase <notifications@github.com> wrote:

> In client/index.html
> <https://github.com/CanteenApp/Canteen/pull/25#discussion_r58943071>:
>
> > @@ -0,0 +1,16 @@
> > +<!DOCTYPE html>
> > +<html lang="en">
> > +<head>
> > +  <meta charset="UTF-8">
> > +  <title>Canteen</title>
> > +  <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" />
>
> I'd rather go with bower... but we could always come back that later and
> just go this route for the time being
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/CanteenApp/Canteen/pull/25/files/16d271461bec16f015bf18ff3097f0d70d23e65f#r58943071>
>